util/doxygen/doxygen.css

```diff
@@ -0,0 +1,1771 @@
+/* The standard CSS for doxygen 1.8.17 */
```

Comment:
If this is unspecified doesn't the default get used anyway?

Reply:
Yeah, but I was close to trying to fit this into the OT Docs Layout. That was kind-of next on my list.